egs2/libriheavy_small/asr1/local/data.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+stage=1
+stop_stage=5000
+data_dir=data
+
+log "$0 $*"
+. utils/parse_options.sh
+
+# url for the official repo.
+libriheavy_repo=https://github.com/k2-fsa/libriheavy.git
+
+if [ $# -ne 0 ]; then
+    log "Error: No positional arguments are required."
+    exit 2
+fi
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+if [ ! -e "${LIBRILIGHT}" ]; then
+    log "Fill the value of 'LIBRILIGHT' of db.sh"
+    exit 1
+fi
+
+if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
+    if [ -d "${LIBRILIGHT}/small" ] && [ -d "${LIBRILIGHT}/medium" ] && [ -d "${LIBRILIGHT}/large" ]; then
+        log "Libri-light found in ${LIBRILIGHT}."
+        rm -fr libriheavy
+        git clone $libriheavy_repo
+    else
+        echo "Some of ${LIBRILIGHT}/{small,medium,large} directories do not exist."
+        echo "Please follow this link and download the audio data to ${LIBRILIGHT}:"
+        echo "https://github.com/facebookresearch/libri-light/tree/main/data_preparation#1a-downloading"
+        exit 1
+    fi
+fi
+
+if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
+    log "Downloading and extracting the Libriheavy manifests"
+    pushd libriheavy
+    bash run.sh --stage 1 --stop-stage 2
+    popd
+fi
+
+if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
+    log "Preparing data directories in ${data_dir}"
+
+    mkdir ${data_dir}
```

Comment:
```suggestion
    mkdir -p ${data_dir}
```
Add options for existing dir

Reply:
Thank you!! From what I see from the README.md under egs2, libriheavy_{small and medium} for ASR are already there. The only one missing is libriheavy_large. 

Please let me know if I am confusing with something else.

Reply:
Do you want me to add more things here for the dir, or the thing you changed is good enough? For me it looks already nice just with the -p. Thanks!

Reply:
I believe just add -p would be fine to prevent it from throwing error in the middle if the folder is already there.